templates/editor/boot-partial.html

```diff
@@ -14,5 +14,15 @@
         }
       };
     {% endif %}
+
+    if (!window.onPiskelReady) {
+      window.onPiskelReady = [];
+    }
+
+    window.onPiskelReady.push(function() {
```

Comment:
Do you think we need that?
It seems impossible because of the script execution order. The boot-partial script will run before Piskel no matter what.